clio/polymorphic_integer_type
# Add compatibility for Rails 6.0 and remove support for Rails 4.2

To add Rails 6.0 compatibility for `polymorphic_integer_type`, we need to:

- Bump the `activerecord` gemspec dependency range to include 6.0
- Add a gemfile/tests for Rails 6-0-stable
- Update the call to `ActiveRecord::MigrationContext`'s initializer as the arguments have changed in 6.0
- Remove calls to `update_attributes` in the specs as `update_attributes` is deprecated in Rails 6.1 (not required, but might as well handle it here)

On top of this, since Rails 4.2 is no longer supported by Rails, I'm removing 4.2 support for all subsequent versions of `polymorphic_integer_type`.

**Note**: While we don't have automated tests yet, running `rake test:all` passed locally.
